pkgs/build-support/vm/test.nix

```diff
@@ -9,20 +9,23 @@ with vmTools;
 
   buildHelloInVM = runInLinuxVM hello;
 
-  buildPanInVM = runInLinuxVM pan;
+  buildPcmanrmInVM = runInLinuxVM (pcmanfm.overrideAttrs (old: {
```

Comment:
Pan would still go oom on 64 core machines, so I just replaced it with a faster build.